design-toolkit/components/src/components/select/styles.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2025 Hypergiant Galactic Systems Inc. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import { tv } from '../../lib/utils';
+
+export const SelectStyles = tv({
+  slots: {
+    select: 'group/select',
+    field: [
```

Comment:
Can you rename this to `trigger` please?

`select` could (should?) be `field`

And Select should be SelectField for consistency. Any time we have a field type component (label, input, helper text, error, etc) it should include that term in the name